presto-main/src/main/java/io/prestosql/operator/aggregation/AggregationFromAnnotationsParser.java

```diff
@@ -119,7 +120,7 @@ public static ParametricAggregation parseFunctionDefinition(Class<?> aggregation
         }
 
         checkArgument(stateSerializerFactories.size() == 1,
-                String.format(
+                format(
```

Comment:
Remove this format call?

Reply:
i didn't notice this one (lack of "String." is due to static importing elsewhere in the file). IntelliJ regex search does not help when `checkArgument` and `format(` are on separate lines.

Fixed in https://github.com/prestosql/presto/pull/217